docs/slate.md

```diff
@@ -12,7 +12,7 @@ $ npm i --save @react-page/plugins-slate
 
 <p>
   <figure align="center">
-    <img alt="Text editing plugin" src="./images/text-editing-plugin.gif"><br>
+    <img alt="Text editing plugin" src="../docs-images/text-editing-plugin.gif"><br>
```

Comment:
thx, I did some linking magic to make the docs run inside the example nextjs app, so i broke some links.

I'll have to check whether it works after deployment.